src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebefore/Example2.java

```diff
@@ -0,0 +1,25 @@
+/*
+NoWhitespaceBefore
+allowLineBreaks = true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespacebefore;
+
+import com.google.common.collect.Lists;
+
+class Example2 {
+  int[][] array = { { 1, 2 }
```

Comment:
If we have agreement on use of line comments to restrict generated example content, should we be placing comments in these files now? Even if we change format, we can `sed` existing comments to conform.

Reply:
I thought we agreed now just do bulk conversation of xdoc to real tests  - Examples.
On second phase: put content of Examples to xdoc by Parser, so we can introduce start-end comments can be introduced as we finish Parser

Reply:
Done. Added comments.